sphinx/writers/latex.py

```diff
@@ -446,6 +446,18 @@ def __init__(self, document, builder):
             self.elements['multilingual'] = '%s\n\\setmainlanguage{%s}' % \
                 (self.elements['polyglossia'], self.babel.get_language())
 
+        # detect Chinese language, disable babel/polyglossia
```

Comment:
OK I'll check the code later.

And about the PR to stable branch. In this version the latex builder/writer does not overrides conf.py so I think it is compatible.
I'll think about where to put `usepackage` and if I add a new conf key then maybe it is better to PR to master branch